test/unit/plugins/rest_spec.js

```diff
@@ -678,20 +682,31 @@ describe("The Rest plugin", function () {
 					return instances.getInstance({ id : instance.id });
 				})
 				.then(function (instance) {
+					terminateInstancesStub = Sinon.stub(awsAdapter, "terminateInstances").returns(
+						Bluebird.resolve({
+							id       : instance.id,
+							ami      : instance.ami,
+							type     : instance.type,
+							state    : "terminated",
+							uri      : instance.uri,
+							revision : instance.revision + 2
+						})
+					);
+
 					updatedInstance = new Instance({
 						id       : instance.id,
 						ami      : instance.ami,
 						type     : instance.type,
-						state    : "failed",
+						state    : "terminated",
 						uri      : instance.uri,
-						revision : instance.revision + 1
+						revision : instance.revision + 2
```

Comment:
If it doesn't really matter than would it be more correct to check for `response.revision >= instance.revision + 1`?